app/views/embedded/petitions/show.html.slim

```diff
@@ -0,0 +1,63 @@
+doctype html
+html
+  body.petition-widget
+    div.petition-container class="#{params[:size] || "big"}"
+      div.petition-header style="background-image: url('#{cycle.picture.url}')"
+        div.petition-header-overlay
+        div.petition-header-info
+          h3= phase.name
+          p= phase.description
+          = image_tag "watermark"
+      div.petition-progress-container
```

Comment:
It would be cool to add something like `8 of 15 signatures` on the `title` here to help the user know that this isn't related to the period and show the signatures count even on the small version.